File: app/src/main/java/com/kelsos/mbrc/mappers/AlbumMapper.java
package com.kelsos.mbrc.mappers;

import com.kelsos.mbrc.data.library.Album;
import com.kelsos.mbrc.domain.AlbumInfo;

public class AlbumMapper implements Mapper<Album,AlbumInfo> {
  @Override
  public AlbumInfo map(Album album) {
    return AlbumInfo.builder().album(album.getAlbum()).artist(album.getArtist()).build();
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/mappers/AlbumMapper.kt
package com.kelsos.mbrc.mappers

import com.kelsos.mbrc.data.library.Album
import com.kelsos.mbrc.domain.AlbumInfo

class AlbumMapper : Mapper<Album, AlbumInfo> {
  override fun map(from: Album): AlbumInfo {
    return AlbumInfo(from.album!!, from.artist!!, from.cover)
  }
}
